config/file_config.go

```diff
@@ -53,8 +53,9 @@ func (r *RulesBasedSamplerConfig) String() string {
 }
 
 type configContents struct {
-	ListenAddr         string        `validate:"required"`
-	PeerListenAddr     string        `validate:"required"`
+	ListenAddr         string `validate:"required"`
```

Comment:
go fmt?

Reply:
I too thought the formatting was wrong. but running `go fmt` doesn't change it. I think it's because the `GRPCListenAddr` entry below doesn't have a descriptor which seems to reset the spacing.